### PR TITLE
8.0 stock lot supplier importer

### PR DESCRIPTION
**Pull request by ismaelcj**
I propose a new module. It allows to import Serial Number (lots) by the supplier product_code (SKU - Stock-Keeping Unit) from a xls file.

The xls needs, at least, the columns 'sku_no' and 'serial_no' (no matter whether it is case sensitive). The program will search the file sku columns on 'Supplier Product Code' field from product supplier info.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
What is the purpose of importing lots, but not quantities?

Thanks for the tests! :wink:

**Comment by ismaelcj** [comment]
The purpose is to have the lots created before processing the picking in. The supplier give us a list of serial number associated with a sku number before send the products. So we have not to read the serial numbers of each product when they come.

File: stock_lot_supplier_importer/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2015 FactorLibre - Ismael Calvo <ismael.calvo@factorlibre.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock Lot Importer by Supplier",
+    "summary": "Import Lots from xls by product's SKU or name, from supplier.",
+    "version": "8.0.1.0.0",
+    "category": "arehouse Management",
```
Comment (pedrobaeza) [inline stock_lot_supplier_importer/__openerp__.py]: s/arehouse/Warehouse

**Comment by pedrobaeza** [comment]
Write the purpose then on the README to make it clear (and please respect 80 cols also on README.rst)

**Comment by pedrobaeza** [comment]
You have to add external_dependencies key with xlrd python dependency on the manifest, and add also on .travis.yml the corresponding package to be installed in order to have Travis happy.

**Comment by gurneyalex** [comment]
@ismaelcj could you rebase the PR in order to trigger a CI rebuild?

**Comment by ismaelcj** [comment]
@gurneyalex, Is it now correct? I don't know why there is three commits...

**Pull request merged**